### ISSUE TITLE
OSD notifies 'GIF start' but nothing happens

### ISSUE THREAD

**Issue by Jogai** [arandomliz/mpv-gif-generator#1]
There is no log generated or a temporary file, and no result gif either. 

- EndeavourOS with ffmpeg4.4 installed

**Comment by tyalie** [comment]
I think I would need a bit more infos. For example what the log says, the medium you tried to capture, ... ^^

**Comment by Jogai** [comment]
Yeah, tought you would say that. But there is nog log, and if I start from terminal the terminal logs nothing either.. Its with all files I tried so far (mp4, avi, mkv)
